views/js/controller/creator/templates/testpart.tpl

```diff
@@ -1,45 +1,36 @@
 <div id="{{identifier}}" class="testpart clearfix">
 
    <h1><span data-bind="identifier">{{identifier}}</span>
-        <div class="actions"> 
+        <div class="actions">
             <div class="tlb">
                 <div class="tlb-top">
                     <span class="tlb-box">
                         <span class="tlb-bar">
                             <span class="tlb-start"></span>
                             <span class="tlb-group">
-                                <a href="#" class="tlb-button-off property-toggler" title="Test Part Properties"><span class="icon-settings"></span></a>
+                                <a href="#" class="tlb-button-off property-toggler" title="{{__ 'Test Part Properties'}}"><span class="icon-settings"></span></a>
                                 <span class="tlb-separator"></span>
-                                <a href="#" class="tlb-button-off move-up" title="Move Up"><span class="icon-up"></span></a>
-                                <a href="#" class="tlb-button-off move-down" title="Move Down"><span class="icon-down"></span></a>
-                                <span class="tlb-separator"></span> 
-                                <a href="#" class="tlb-button-off" title="Remove Test Part" data-delete="#{{identifier}}"><span class="icon-bin"></span></a>
+                                <a href="#" class="tlb-button-off move-up" title="{{__ 'Move Up'}}"><span class="icon-up"></span></a>
+                                <a href="#" class="tlb-button-off move-down" title="{{__ 'Move Down'}}"><span class="icon-down"></span></a>
```

Comment:
I don't see those two keys in the `.po` files.

Reply:
I don't need to edit because these keys already exist. I just update the template. Check line 753 for `Move Up` and line 720 for `Move Down`

Reply:
Right you are. My bad, was searching for them in a case-sensitive manner (providing a wrong case of course).